### DISCUSSION TITLE
auto-convert into webp

### DISCUSSION THREAD

**Discussion by mauspartyhouse** [datafeedr/wordpress-plugins#34] [Help]
Dear all,

I want al images to be present on my website as .webp and Im looking for a solution to automate image convertion. 
The website is an an affiliate one with Datafeedr as product import tool. 

Does anyone has experience with this?
Thank you

**Comment by EricBusch** [comment]
There are some WordPress plugins that will convert images into .webp. I have not used any so I can't recommend any.

**Comment by xtra-dev** [comment]
Use Optimus HQ or HQ Pro - works great: https://optimus.io
